### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by sedited**
This cleans up and consolidates some of the docs. The release process description will need some further refining when it is better defined.

### CODE REVIEW

File: doc/README_osx.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-Deterministic OS X Dmg Notes.
+Cross Plattform and Deterministic OS X Dmg Notes.
+================================================
 
 Working OS X DMGs are created in Linux by combining a recent clang,
 the Apple binutils (ld, ar, etc) and DMG authoring tools.
```
Comment (denravonska) [inline doc/README_osx.md:5]: Maybe add some like "or by using homebrew and the [git-jiro](https://github.com/Git-Jiro/homebrew-jiro) recipe".

Edit: oh nvm, this is only about cross platform.

File: doc/release-process.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+    git shortlog --no-merges v(current version, e.g. 0.7.2)..v(new version, e.g. 0.8.0)
+
+Generate list of authors:
+
+    git log --format='%aN' "$*" | sort -ui | sed -e 's/^/- /'
+
+Tag version (or release candidate) in git
+
+    git tag -s v(new version, e.g. 0.8.0)
```
Comment (denravonska) [inline doc/release-process.md:38]: We don't use the "v" prefix in the 3.x tags.

**Comment by denravonska** [comment]
Great work. Only skimmed through the gitian doc as I don't have any knowledge to say if it's correct or not.

This closes #805.

File: doc/release-process.md (hunk 2 above)
Comment (sedited) [reply doc/release-process.md:38]: Yes, this should already be reflected in the actual gitian script. I'll change it, when I'm able to compile an actual gitian.

**Comment by sedited** [comment]
Removed the qt4 build instructions, since it is no longer supported.

**Pull request merged**